### ISSUE TITLE
Invisible entities with names cause a crash on world or server join

### ISSUE THREAD

**Issue by TooterTutor** [Incandescent-Turtle/FriendHighlighter#13]
I launched with the 2.0.0b version on 1.20.1 and joined a server and immediately crashed due to rendering an invisible armor stand. I've posted the crash report and details below.
```
---- Minecraft Crash Report ----
// Don't do that.

Time: 2024-12-28 07:13:17
Description: Rendering entity in world

org.spongepowered.asm.mixin.injection.throwables.InjectionError: LVT in net/minecraft/class_897::method_3926(Lnet/minecraft/class_1297;Lnet/minecraft/class_2561;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;I)V has incompatible changes at opcode 20 in callback friendhighlighter.mixins.json:EntityRendererMixin from mod friendhighlighter->@Inject::captureEntity(Lnet/minecraft/class_1297;Lnet/minecraft/class_2561;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;IFLorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V.
 Expected: [Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;]
    Found: []
Available: []
	at net.minecraft.class_897.handler$ein000$friendhighlighter$captureEntity$missing(class_897.java)
	at net.minecraft.class_897.method_3926(class_897.java)
	at net.minecraft.class_897.method_3936(class_897.java:72)
	at net.minecraft.class_922.method_4054(class_922.java:150)
	at net.minecraft.class_922.method_3936(class_922.java:29)
	at net.minecraft.class_898.method_3954(class_898.java:145)
	at net.minecraft.class_761.method_22977(class_761.java:1574)
	at net.minecraft.class_761.method_22710(class_761.java:1324)
	at net.minecraft.class_757.md8c189f$anaglyph-fabric$lambda$renderWorldTwice$0$1(class_757.java:3113)
	at vlad2305m.anaglyphfabric.client.Util.renderTwice(Util.java:44)
	at net.minecraft.class_757.redirect$zie000$anaglyph-fabric$renderWorldTwice(class_757.java:3113)
	at net.minecraft.class_757.method_3188(class_757.java:1110)
	at net.minecraft.class_757.method_3192(class_757.java:880)
	at net.minecraft.class_310.method_1523(class_310.java:1219)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_897.handler$ein000$friendhighlighter$captureEntity$missing(class_897.java)
	at net.minecraft.class_897.method_3926(class_897.java)
	at net.minecraft.class_897.method_3936(class_897.java:72)
	at net.minecraft.class_922.method_4054(class_922.java:150)
	at net.minecraft.class_922.method_3936(class_922.java:29)
	at net.minecraft.class_898.method_3954(class_898.java:145)
	at net.minecraft.class_761.method_22977(class_761.java:1574)
	at net.minecraft.class_761.method_22710(class_761.java:1324)
	at net.minecraft.class_757.md8c189f$anaglyph-fabric$lambda$renderWorldTwice$0$1(class_757.java:3113)
	at vlad2305m.anaglyphfabric.client.Util.renderTwice(Util.java:44)
	at net.minecraft.class_757.redirect$zie000$anaglyph-fabric$renderWorldTwice(class_757.java:3113)
	at net.minecraft.class_757.method_3188(class_757.java:1110)
``` 

Here are the details of the entity that was trying to be rendered:
```
-- Entity being rendered --
Details:
	Entity Type: minecraft:armor_stand (net.minecraft.class_1531)
	Entity ID: 226661
	Entity Name: ? Elevator ?
	Entity's Exact location: -1863.50, 64.00, 3666.50
	Entity's Block location: World: (-1864,64,3666), Section: (at 8,0,2 in -117,4,229; chunk contains blocks -1872,-64,3664 to -1857,319,3679), Region: (-4,7; contains chunks -128,224 to -97,255, blocks -2048,-64,3584 to -1537,319,4095)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
```

I'll do further testing, but it appears `captureEntity` in the EntityRenderMixin Mixin is the culprit.

Edit: It appears it crashes with armor stands that are invisible and have a custom display name.

**Comment by Incandescent-Turtle** [comment]
Ah thank you, I think this might be an issue with the port to 1.20.1 I did. Cause this sounds familiar. I will investigate ty

**Comment by TooterTutor** [comment]
No problem. Thanks for the response!

**Comment by Incandescent-Turtle** [comment]
This should be fixed as of v2.0.1b, https://github.com/Incandescent-Turtle/FriendHighlighter/releases/tag/v2.0.1beta-1.20.1

The issue was with a method signature change between 1.21 and 1.20.1 for the `EntityRenderer` method `renderLabelIfPresent`. `float tickDelta` was added in 1.21. I forgot to update this when i ported to 1.20.1
